widgets/index.rst

```diff
@@ -0,0 +1,32 @@
+Esportare i dati dai widget
+--------------------------------
+
+.. highlights::
+
+   Come esportare i dati di un widget
+
+Esportazione dei dati
+~~~~~~~~~~~~~~~~~~~~~
+Alcuni widget della dashboard della sezione analytics,
```

Comment:
solo alcuni? Ricordavo si potessero esportare in tutti

Reply:
Diciamo il 90% sì.
Ad esempio, che non hanno questa feature:
- visite in tempo reale, si deve andare sul log delle visite
- mappa delle visite, si deve andare nella sezione dei dettagli ed estrarre i dati da quelli che mostrano i grafici per continente, nazione, etc.
- in generale quelli che mostrano i mini-grafici ("overview") no, bisogna andare nei dettagli

Reply:
Allora scriviamo "la maggior parte dei widget"